.gitignore

```diff
@@ -14,6 +16,7 @@ pids
 *.pid
 *.seed
 *.pid.lock
+package-lock.json
```

Comment:
The package-lock.json shouldn't be in .gitignore - remove and double check that the file is then not deleted on the PR! (hence the giant -4684 lines in this PR)